franklin/core/views.py

```diff
@@ -81,17 +21,3 @@ def health(request):
             required: true
     """
     return Response(status=status.HTTP_200_OK)
-
-
-@partial
-def save_oauth(strategy, details, user=False, *args, **kwargs):
-    # TODO django-rest-framework-social-oauth2 has a bug in it (11/2015)
-    # It doesn't save the access_token, so we do it here manually
-    request = kwargs.get('request', None)
-    if user and request:
-        access_token = request.get('token', None)
-        social = user.social_auth.get(provider='github')
-        social.extra_data['access_token'] = access_token
-        social.save()
-        return {'is_new': False, 'user': user}
-    return {'is_new': True, 'user': None}
```

Comment:
This bug still exists, but I've moved this code into the authentication middleware. 
My current theory is that this bug occurs because of the library version franklin is using. I'm fairly certain access_token was being saved for us at lone point, but then broken after we updated library versions. Will continue to investigate and may end up opening a PR into python-social-auth.